### PR TITLE
circular-passionfruit: Remove "Resume Coding" button and add "Edit Project" to recent projects project items

### PR DESCRIPTION
**Pull request by taravancil**
## Links
* [circular-passionfruit.glitch.me](https://circular-passionfruit.glitch.me/)
* [#453](https://github.com/FogCreek/glitch-community-work/issues/453)
* [User dashboard spec](https://docs.google.com/document/d/10KS93TQ6vaQcVSL0ujUj4__GGl7M5MlS1vimzNqiizU/edit#heading=h.3im6lap0txtw)

## GIF/Screenshots:
Screen Shot 2019-10-03 at 11 23 55 AM

## Changes:
* Remove the "Resume Coding" button
* Add the `showEditButton` prop to `<ProjectsList>` and `<ProjectItem>`, which if true, will render a footer with an "Edit Project Button"

## How To Test:
* Log in, you should see the "Edit Project" button on your recent projects dashboard on the homepage
* The "Resume Coding" should be gone
* Check out other `<ProjectsList>`s, you should not see the "Edit Project" button on those `<ProjectItem>`s
  - Team page recent projects
  - User page recent projects

### CODE REVIEW

**Comment by taravancil** [comment]
@ttseng to confirm: the "Edit Project" button should only be on project items in the user dashboard, right?

**Comment by taravancil** [comment]
> I would really want this on any project I have the ability to edit, though...

Maybe! Project lists often display dozens of projects at a time - it's not clear to me if it would be odd to have the button displayed dozens of times on a page. I'll leave the decision to Tiff!

File: src/components/project/project-item.styl [hunk 1]
```diff
@@ -127,4 +129,9 @@ a.linkBody:hover
     
 .privateIcon
   margin: 4px 5px 0
-  color: private-foreground
\ No newline at end of file
+  color: private-foreground
+  
+.footer
+  background: secondary
```
Comment (ttseng) [inline src/components/project/project-item.styl]: do you mind changing this to `secondary-background` instead? That way the project info is the primary thing that your eye is drawn to, and the edit project appears secondary: 
image

**Comment by ttseng** [comment]
let's see what the reaction is to just user dashboard changes, and then determine whether it's needed everywhere.  

having them default link to project pages everywhere else just may be the only way where project authors discover we have project pages - and that their READ.mes appear there 😂

**Comment by taravancil** [comment]
Updated the footer background to `secondary-background`

Screen Shot 2019-10-03 at 3 07 01 PM

**Comment by taravancil** [comment]
> is the idea to add this in advance of the rest of the dashboard work?

yep, if that's ok with you!

**Pull request merged**